src/main/java/Main.java

```diff
@@ -33,8 +33,11 @@ public static void main(String[] args) throws IOException, ClassNotFoundExceptio
         JPanel screens = new JPanel(cardLayout);
         application.add(screens);
 
+        //create readwriter - read in TaskMap from file upon program start
+        TaskReadWrite taskReadWrite = new TaskReadWrite("src/main/java/data/TaskMap.txt");
+        TaskMap.load(taskReadWrite);
```

Comment:
To address your first comment: I ended up making a small adjustment in the load method itself (in TaskMap) so that the map is initialized to a blank HashMap if reading from a file throws an exception. I did this because it was printing an error message before when the map was getting initialized to null.

Also, all of the files and refactoring were a complete mistake from trying to merge changes and push commits quickly! That's 100% my bad